packages/devtools-reps/src/object-inspector/utils/client.js

```diff
@@ -80,6 +80,19 @@ async function getPrototype(
   return objectClient.getPrototype();
 }
 
+async function getFullText(
+  objectClient: ObjectClient,
+  stringGrip: Node,
+) : ?Promise<{prototype?: Object}> {
```

Comment:
I think this should be something like `: ?Promise<{_fullText?: string}>`

Reply:
👍